src/cljs/lumo/classpath.cljs

```diff
@@ -0,0 +1,45 @@
+(ns lumo.classpath
+  (:require [clojure.string :as string]))
+
```

Comment:
We also need docstrings in the public functions so that users can use the `doc` REPL special to check their behavior